shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -49,6 +49,19 @@
     accessibility_bridge_.reset();
```

Comment:
should we get rid of that observer here too?

Reply:
We are getting rid of the observer a couple lines down, line 55.  I don't believe there is any reason why it should be happening higher in the method.